app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+    def index 
```

Comment:
probably rubocop shows that as a mistake 
in other files, you uses 2 spaces but here you have 4 spaces